diff --git a/node_modules/electron-installer-redhat/resources/spec.ejs b/node_modules/electron-installer-redhat/resources/spec.ejs
index e07a8a3..0b3f447 100644
--- a/node_modules/electron-installer-redhat/resources/spec.ejs
+++ b/node_modules/electron-installer-redhat/resources/spec.ejs
@@ -24,7 +24,7 @@ print('\n\n\n') }
 
 %>%install
 mkdir -p %{buildroot}/usr/
-cp <%= process.platform === 'darwin' ? '-R' : '-r' %> usr/* %{buildroot}/usr/
+cp <%= process.platform === 'darwin' ? '-R' : '-r' %> <%= usrPath %> %{buildroot}/usr/
 
 
 %files
diff --git a/node_modules/electron-installer-redhat/src/dependencies.js b/node_modules/electron-installer-redhat/src/dependencies.js
index 0b583b9..1c5ddf7 100644
--- a/node_modules/electron-installer-redhat/src/dependencies.js
+++ b/node_modules/electron-installer-redhat/src/dependencies.js
@@ -33,6 +33,13 @@ async function rpmSupportsBooleanDependencies (logger) {
   return rpmVersionSupportsBooleanDependencies(await getRpmVersion(logger))
 }
 
+/**
+ * Retrieves the RPM version number and determines the path to the `usr` directory based on the version.
+ */
+async function getRpmUsrPath (logger) {
+  return rpmUsrPath(await getRpmVersion(logger))
+}
+
 async function getRpmVersion (logger) {
   const versionOutput = await spawn('rpmbuild', ['--version'], logger)
   return _.last(versionOutput.trim().split(' '))
@@ -48,6 +55,14 @@ function rpmVersionSupportsBooleanDependencies (rpmVersionString) {
   return rpmVersion >= [4, 13, 0]
 }
 
+/**
+ * Determines the path to the `usr` directory based on the RPM version.
+ */
+function rpmUsrPath (rpmVersionString) {
+  const rpmVersion = rpmVersionString.split('.').slice(0, 3).map(n => parseInt(n))
+  return rpmVersion >= [4, 20, 0] ? '../usr/.' : 'usr/*'
+}
+
 /**
  * Transforms the list of trash requires into an RPM boolean dependency list.
  */
@@ -75,6 +90,7 @@ module.exports = {
     }
   },
   getRpmVersion,
+  getRpmUsrPath,
   rpmSupportsBooleanDependencies,
   rpmVersionSupportsBooleanDependencies,
   trashRequiresAsBoolean
diff --git a/node_modules/electron-installer-redhat/src/installer.js b/node_modules/electron-installer-redhat/src/installer.js
index c7a3dbb..59d6a13 100644
--- a/node_modules/electron-installer-redhat/src/installer.js
+++ b/node_modules/electron-installer-redhat/src/installer.js
@@ -87,6 +87,7 @@ class RedhatInstaller extends common.ElectronInstaller {
     ])
     this.defaults = {
       ...common.getDefaultsFromPackageJSON(pkg, { revision: 1 }),
+      usrPath: await redhatDependencies.getRpmUsrPath(this.userSupplied.logger),
       version: pkg.version || '0.0.0',
       license: pkg.license,
       compressionLevel: 2,
